bot/src/authorisation.ts

```diff
@@ -0,0 +1,69 @@
+import getSigner from 'auctions-core/src/signer';
+import {
+    authorizeCollateral,
+    authorizeWallet,
+    getCollateralAuthorizationStatus,
+    getWalletAuthorizationStatus,
+} from 'auctions-core/src/authorizations';
+import { ETHEREUM_NETWORK, KEEPER_PREAUTHORIZE } from './variables';
+import { getWhitelistedCollaterals } from './whitelist';
+
+export async function checkAndAuthorizeWallet() {
```

Comment:
I'm not sure why typescript return type is not required here and in other functions 🙁